### ISSUE TITLE
Text can't be selected when using simple theme with JS enabled

### ISSUE THREAD

**Issue by chayleaf** [searxng/searxng#794]
(Commit: 96ab5e57ff92aba4ab0fedeb4e5fbfa09e9fc2e2,  most likely reproducible on master)

When using the "simple" theme, try selecting any text in the search results. It won't get selected, presumably because of a custom selection event handler. Said event handler should not prevent text from being selected (For example, personally I rely on being able to do that a lot). Also, text can be selected with JS disabled.

**Comment by return42** [comment]
Is this related to https://github.com/searxng/searxng/issues/778 ? .. would your issue resolved by this suggestion https://github.com/searxng/searxng/issues/778#issuecomment-1016593816 ?

**Comment by not-my-profile** [comment]
No this issue is not about the autocompletion, it's about the actual results.

**Comment by return42** [comment]
@pavlukivan sorry for the delay .. could you please give it a try; I prepared PR https://github.com/searxng/searxng/pull/997 to fix the issue.

**Comment by mrpaulblack** [comment]
The bug fix for this issue contained another bug: #1002. Because of that the bug fix got reverted on master for now until we have a proper fix; Reopening.

**Comment by chayleaf** [comment]
~~as of commit 36184b2f29187e37061590b40ce404b6a2a20c55, I still can't select text~~

edit: nvm the script was cached, it works properly now!

**Comment by return42** [comment]
> nvm the script was cached,

This is a pitfall I also often tap .. thanks for verifying :+1:
